docs/core/learn-more/authentication.md

```diff
@@ -1,35 +1,97 @@
-# Authentication
+# Authentication and Authorization
 
-## Introduction
+## Authentication
+
+Chain Core allows control of which entities have access to certain features of the system. There are two methods available for limiting access:
+
+1. Access tokens using HTTP Basic Authentication
+2. x509 client certificates
```

Comment:
Nit: it's spelled "X.509" in actual text (x509 is when we're being lazy or as a variable name).